src/tools/schema2kotlin.ts

```diff
@@ -40,14 +40,20 @@ export class Schema2Kotlin extends Schema2Base {
   }
 
   fileHeader(outName: string): string {
+    const withCustomPackage = (populate: string) => this.scope !== 'arcs' ? populate : '';
     return `\
 package ${this.scope}
 
 //
 // GENERATED CODE -- DO NOT EDIT
 //
 // Current implementation doesn't support references or optional field detection
-`;
+
+${withCustomPackage(`import arcs.Particle;
+import arcs.Entity; 
+import arcs.StringDecoder;
+import arcs.StringEncoder; 
```

Comment:
No semicolons in Kotlin :upside_down_face:

Reply:
Also, imports should be alphabetically sorted.

Reply:
Man, I get really confusing rapidly switching from typescript to kotlin. Thanks for catching this.